agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -180,10 +187,9 @@ static bool fill_platform_settings(
         return false;
     }
 
-    // set local_gw flag
-    platform_common_conf.local_gw =
-        (platform_common_conf.operating_mode == BPL_OPER_MODE_GATEWAY ||
-         platform_common_conf.operating_mode == BPL_OPER_MODE_GATEWAY_WISP);
+    // Set local_gw flag
+    db->device_conf.local_gw = (platform_common_conf.operating_mode == BPL_OPER_MODE_GATEWAY ||
+                                platform_common_conf.operating_mode == BPL_OPER_MODE_GATEWAY_WISP);
```

Comment:
```suggestion
    // Set local_gw flag
    auto operating_mode = platform_common_conf.operating_mode;
    db->device_conf.local_gw = (operating_mode  == BPL_OPER_MODE_GATEWAY || operating_mode  == BPL_OPER_MODE_GATEWAY_WISP);
```

Reply:
What does it give to define an extra variable and copy the value to it? Nothing.